README.md

```diff
@@ -1,4 +1,4 @@
-# Algernon Launcher [![GitHub release](https://img.shields.io/github/release/hpi-swa-teaching/Algernon-Launcher.svg?label=small%20release&maxAge=0)](https://github.com/hpi-swa-teaching/Algernon-Launcher/releases/latest) [![Github All Releases](https://img.shields.io/github/downloads/hpi-swa-teaching/Algernon-Launcher/total.svg?maxAge=0)](https://github.com/hpi-swa-teaching/Algernon-Launcher/releases) ![smalltalkCI](https://github.com/hpi-swa-teaching/Algernon-Launcher/workflows/smalltalkCI/badge.svg)
+# Algernon Launcher [![GitHub release](https://img.shields.io/github/release/hpi-swa-teaching/Algernon-Launcher.svg?label=small%20release&maxAge=0)](https://github.com/hpi-swa-teaching/Algernon-Launcher/releases/latest) [![Github All Releases](https://img.shields.io/github/downloads/hpi-swa-teaching/Algernon-Launcher/total.svg?maxAge=0)](https://github.com/hpi-swa-teaching/Algernon-Launcher/releases) [![Build Status](https://img.shields.io/travis/hpi-swa-teaching/Algernon-Launcher/master.svg?maxAge=0)](https://travis-ci.org/hpi-swa-teaching/Algernon-Launcher) [![Coverage Status](https://img.shields.io/coveralls/hpi-swa-teaching/Algernon-Launcher/master.svg?maxAge=0)](https://coveralls.io/github/hpi-swa-teaching/Algernon-Launcher?branch=master)
```

Comment:
Is there a reason you added the code coverage badge again? Or is this a merge problem?

Reply:
No seems to be a merge problem